content/posts/April-2021.md

```diff
@@ -28,3 +28,18 @@ weight: 76
   ([packit-service#1046](https://github.com/packit/packit-service/pull/1046)).
   This way users will receive a more immediate feedback about the Service
   handling their requests.
+
+## Week 16 (April 19th - April 23th)
+
+- The `current_version_command` and `create_tarball_command` config options are being deprecated
+  in favour of [actions](https://packit.dev/docs/actions/).
+  The issue will be created in the affected repositories once we find those options in use.
```

Comment:
- `The` suggests some specific issue
- `once` suggests we are actively searching the repositories for such options, which is not the case, is it?

```suggestion
  An issue will be created in the affected repositories if we find those options in use.
```

Reply:
You are right. We are not actively searching. We create an issue during the processing of some event.